docs/docs/libraries/README.md

```diff
@@ -0,0 +1,22 @@
+# IOTA Wallet Libraries
+
+There are currently available the following official bindings to `Wallet.rs`:
+- [Rust](rust/README.md) 
```

Comment:
For Rust it's the source and not a binding, but we have a Java binding

Reply:
I've rephrased this, but I think we will need to create the whole libraries/Java section.  @huhn511 do you think it'd be good idea to work on that now or is it out of scope?